plugins/woocommerce/client/blocks/assets/js/blocks/product-filters/inner-blocks/chips/style.scss

```diff
@@ -17,7 +17,20 @@
 	font-size: 0.875em;
```

Comment:
I would update these values too. Setting the `font-size` to `0.8em` and the `padding` to `.25em .75em` to match the _Add to Cart + Options_ block.